src/js/utils/styles.js

```diff
@@ -4,7 +4,7 @@ import { palm, parseMetricToNum } from './mixins';
 import { colorForName, colorIsDark, getRGBA } from './colors';
 
 export const activeStyle = css`
-  background-color: ${props => props.theme.global.hover.backgroundColor};
+  background: ${props => props.theme.global.hover.backgroundColor};
```

Comment:
I'm wondering if we should change this to `global.hover.background`, like I did for Layer, to allow more flexibility and to be more consistent?

Reply:
can we do this in a separate pr ?

Reply:
Yes.